examples/twittermap/web/public/javascripts/map/pointmap.js

```diff
@@ -158,66 +158,69 @@ angular.module('cloudberry.map')
         $scope.scale_x = 0;
         $scope.scale_y = 0;
 
-        //To generate Tweet Popup content from Twitter API (oembed.json?) response JSON
-        function translateOembedTweet(tweetJSON) {
-          var userName = "";
-          try {
-            userName = tweetJSON.author_name;
-          }
-          catch (e){
-            console.log("author_name missing in this Tweet.:" + e.message);
-          }
+        //shows each point's info in Front-end.
+        function translateTweetdatatoShow(tweetJSON) {
+            var userName = "";
+            try {
+                userName = tweetJSON['user.name'];
+            }
+            catch (e){
+                console.log("author_name missing in this Tweet. :" + e.message);
+            }
 
-          var userLink = "";
-          try {
-            userLink = tweetJSON.author_url;
-          }
-          catch (e) {
-            console.log("author_url missing in this Tweet.:" + e.message);
-          }
+            var userPhotoUrl = "";
+            try {
+                userPhotoUrl= tweetJSON['user.profile_image_url'];
+                console.log("userPhotoUrl = " + userPhotoUrl);
+            }
+            catch (e){
+                console.log("user.profile_image_url missing in this Tweet.:");
+            }
 
-          var tweetLink = "";
-          try {
-            tweetLink = tweetJSON.url;
-          }
-          catch (e){
-            console.log("url missing in this Tweet.:" + e.message);
-          }
+            var tweetText = "";
+            try {
+                tweetText = tweetJSON.text;
+            }
+            catch (e){
+                console.log("Text missing in this Tweet. :" + e.message);
+            }
+            if (tweetText === "" || null || undefined)
+            {
+                tweetText = "Fail to get Tweets data."
+            }
 
-          var tweetText = "";
-          try {
-            var tweetHtml = new DOMParser().parseFromString(tweetJSON.html, 'text/html');
-            tweetText = tweetHtml.getElementsByTagName('p')[0].innerHTML;
-          }
-          catch (e){
-            console.log("html missing in this Tweet.:" + e.message);
-          }
+            var tweetTime = "";
+            try {
+                var created_at = new Date(tweetJSON['create_at']);
+                tweetTime = created_at.toTimeString() + "\\t" + created_at.toDateString();
```

Comment:
From the screenshot, it seems this \\t doesn't make any effect there. Try another way to fill the space, e.g. html tag for whitespace is "&nbsp"